apps/ledger-live-desktop/jest.config.js

```diff
@@ -37,6 +37,9 @@ const defaultConfig = {
 };
 
 module.exports = {
+  collectCoverage: true,
+  collectCoverageFrom: ["src/**/*.{ts,tsx}"],
```

Comment:
I think that we can exclude test and story files =>
```suggestion
  collectCoverageFrom: ["src/**/*.{ts,tsx}","!src/**/*.test.{ts,tsx}","!src/**/*.stories.{ts,tsx}"],
```
Maybe we have other patterns to exclude.